Examples/src/main/java/FilterImage.java

```diff
@@ -0,0 +1,111 @@
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ *  SOFTWARE.
+ * </summary>
+ * --------------------------------------------------------------------------------------------------------------------
+ */
+
+import com.aspose.imaging.cloud.sdk.model.BigRectangularFilterProperties;
+import com.aspose.imaging.cloud.sdk.model.FilterPropertiesBase;
+import com.aspose.imaging.cloud.sdk.model.requests.FilterEffectImageRequest;
+
+/**
+ * Apply a filtering effect to an image.
+ */
+public class FilterImage extends ImagingBase {
+    /**
+     * Initializes a new instance of the FilterImage class
+     *
+     * @param imagingApi The imaging API.
+     */
+    public FilterImage(com.aspose.imaging.cloud.sdk.api.ImagingApi imagingApi) {
+        super(imagingApi);
+        printHeader("Filter image example:");
+    }
+
+    /**
+     * Gets the name of the example image file.
+     *
+     * @return The name of the example image file.
+     */
+    @Override
+    protected String getSampleImageFileName() {
+        return "FilterEffectSampleImage.psd";
+    }
+
+    /**
+     * Applies  filtering effect to an image from cloud storage.
+     *
+     * @throws Exception
+     */
+    public void FilterImageFromStorage() throws Exception {
+        System.out.println("Apply filtering effect to an image from cloud storage");
+
+        uploadSampleImageToCloud();
+
+        String filterType = "BigRectangular";
+        FilterPropertiesBase filterProperties = new BigRectangularFilterProperties();
+        String format = "gif";
+        String folder = CloudPath; // Input file is saved at the Examples folder in the storage
+        String storage = null; // We are using default Cloud Storage
+
+        FilterEffectImageRequest request = new FilterEffectImageRequest(getSampleImageFileName(), filterType,
+                filterProperties, format, folder, storage);
+
+        System.out.println(String.format("Call FilterEffectImage with params: filter type: %s, format: %s",
+                filterType, format));
+
+        byte[] updatedImage = ImagingApi.filterEffectImage(request);
+        saveUpdatedSampleImageToOutput(updatedImage, false, format);
+
+        System.out.println();
+    }
+
+    /**
+     * Applies  filtering effect to an image and uploads to Cloud Storage.
+     *
+     * @throws Exception
+     */
+    public void FilterImageAndUploadToStorage() throws Exception {
+        System.out.println("Apply filtering effect to an image and upload to cloud storage");
+
+        uploadSampleImageToCloud();
+
+        String filterType = "BigRectangular";
+        FilterPropertiesBase filterProperties = new BigRectangularFilterProperties();
+        String format = "gif";
```

Comment:
This filter is useful to reduce noise but not very popular. Would you mind to use Gaussian blur or Gaussian wiener filters as known  better ?